[2023-07-20T14:54:52.820+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.dag_envio_alerta manual__2023-07-20T14:53:46.536980+00:00 [queued]>
[2023-07-20T14:54:52.918+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.dag_envio_alerta manual__2023-07-20T14:53:46.536980+00:00 [queued]>
[2023-07-20T14:54:52.919+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-20T14:54:52.933+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): dag_envio_alerta> on 2023-07-20 14:53:46.536980+00:00
[2023-07-20T14:54:52.937+0000] {standard_task_runner.py:57} INFO - Started process 236 to run task
[2023-07-20T14:54:52.940+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_stock_data_google_DAG', 'dag_envio_alerta', 'manual__2023-07-20T14:53:46.536980+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/ETL_stock_data_google_DAG.py', '--cfg-path', '/tmp/tmpjho0q7me']
[2023-07-20T14:54:52.942+0000] {standard_task_runner.py:85} INFO - Job 99: Subtask dag_envio_alerta
[2023-07-20T14:54:52.986+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_stock_data_google_DAG.dag_envio_alerta manual__2023-07-20T14:53:46.536980+00:00 [running]> on host 90317fb99301
[2023-07-20T14:54:53.058+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fvinciguerra' AIRFLOW_CTX_DAG_ID='ETL_stock_data_google_DAG' AIRFLOW_CTX_TASK_ID='dag_envio_alerta' AIRFLOW_CTX_EXECUTION_DATE='2023-07-20T14:53:46.536980+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-20T14:53:46.536980+00:00'
[2023-07-20T14:54:53.060+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-20T14:54:53.061+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/pyscripts/Alert_monthvalues_stock_data_google.py']
[2023-07-20T14:54:53.068+0000] {subprocess.py:86} INFO - Output:
[2023-07-20T14:54:54.185+0000] {subprocess.py:93} INFO - WARNING:root:/opt/***/logs/scheduler/latest already exists as a dir/file. Skip creating symlink.
[2023-07-20T14:54:56.048+0000] {subprocess.py:93} INFO - [[34m2023-07-20T14:54:56.047+0000[0m] {[34mAlert_monthvalues_stock_data_google.py:[0m61} WARNING[0m - Alerta: Â¡El campo MonthVolume (521386300.00) para 2023-06-30 estÃ¡ por encima de 10000000![0m
[2023-07-20T14:54:59.342+0000] {subprocess.py:93} INFO - Exito al enviar el correo
[2023-07-20T14:54:59.553+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-20T14:54:59.579+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=ETL_stock_data_google_DAG, task_id=dag_envio_alerta, execution_date=20230720T145346, start_date=20230720T145452, end_date=20230720T145459
[2023-07-20T14:54:59.630+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-20T14:54:59.646+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
